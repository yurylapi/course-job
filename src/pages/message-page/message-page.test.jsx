import React from 'react';
import { shallow } from 'enzyme';
import MessagePage from './message-page';

describe('MessagePage component', () => {
    let routeProps;
    let message;

    beforeAll(() => {
        message = 'alert message';
        routeProps = {
            match: {
                params: {
                    filmId: 11111
                }
            }
        };
    });

    it('should be rendered correctly', () => {
        const component = shallow(<MessagePage message={message} {...routeProps} />);
        expect(component).toMatchSnapshot();
    });
});
