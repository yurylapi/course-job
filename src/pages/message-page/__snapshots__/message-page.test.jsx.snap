// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagePage component should be rendered correctly 1`] = `
<Fragment>
  <Connect(SearchForm)
    submitHandler={[Function]}
  />
  <ResultsMessage
    message="alert message"
  />
  <Footer />
</Fragment>
`;
